tests/test_metadata.py

```diff
@@ -262,3 +274,46 @@ def test_tdm_current_records_most_recent_version(timdex_metadata_with_deltas):
             == most_recent.iloc[0]["run_timestamp"]
         )
         assert current_version.iloc[0]["run_id"] == most_recent.iloc[0]["run_id"]
+
+
+def test_tdm_merge_append_deltas_static_counts_match_records_count_before_merge(
+    timdex_metadata_with_deltas, timdex_metadata_merged_deltas
+):
+    static_count_merged_deltas = timdex_metadata_merged_deltas.conn.query(
+        """select count(*) as count from static_db.records;"""
+    ).fetchone()[0]
+    assert static_count_merged_deltas == timdex_metadata_with_deltas.records_count
+
+
+def test_tdm_merge_append_deltas_adds_records_to_static_db(
+    timdex_metadata_with_deltas, timdex_metadata_merged_deltas
+):
+    append_deltas = timdex_metadata_with_deltas.conn.query(
+        f"""
+            select
+            {','.join(ORDERED_METADATA_COLUMN_NAMES)}
+            from metadata.append_deltas
+        """
+    ).to_df()
+
+    merged_static_db = timdex_metadata_merged_deltas.conn.query(
+        f"""
+            select
+            {','.join(ORDERED_METADATA_COLUMN_NAMES)}
+            from static_db.records
+        """
+    ).to_df()
```

Comment:
This is absolutely not a shortcoming of this PR, but just something that is not fully established yet.  

In reality, for each row, the following _should_ be a unique identifier combination (aka a primary key): `(timdex_record_id, run_id)`.  

But, there are a handful of combinations that _should_ be unique, including `(run_id, run_record_offset)`.  

This checking of _all_ columns was surprising to me, as we probably only need to check those columns which make them unique... but it's also really thorough!  We ensure that all columns are reproduced.  

Dig it!